### DISCUSSION TITLE
script Suggestions SHOULD order by match-degree

### DISCUSSION THREAD

**Discussion by jzq740176597** [godotengine/godot-proposals#6649] [Scripting]
### Describe the project you are working on

3d

### Describe the problem or limitation you are having in your project

have to scroll down or seveal extra key-stroke to get the exact match symbol

see image
[https://i.postimg.cc/YqZJRrtC/4-6-2023-11-25-30-AM.jpg](url)

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

1. order the suggesion list by match degree (I can easily to choose the right-one **ON TOP**)
2. Maybe change default choose the first one (like Visual Assitant)

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

*

### If this enhancement will not be used often, can it be worked around with a few lines of script?

no

### Is there a reason why this should be core and not an add-on in the asset library?

core

**Comment by Zireael07** [comment]
Yuri already told you to stop posting low-quality requests.

Autocomplete is known to be buggy/working wrong in many cases and there's already a PR open to improve it [#65655 ](https://github.com/godotengine/godot/pull/65655). You should at least check the main repo and/or other community channels before opening a request, to be sure it's not a bug
